### ISSUE TITLE
More accurate error messages

### ISSUE THREAD

**Issue by RabebOthmani** [microsoftgraph/microsoft-graph-explorer-v4#2429]
There are scenarios where the call fails but the error presented to the user doesn't reflect the issue clearly. 
Example : 
* code snippets showing snippet not available when it should say retry again or fetching.
* permissions tab showing permission not available when it should say retry again or fetching.

Expected behavior: the message should guide the user to the best action they should take.

**Comment by RabebOthmani** [comment]
@adhiambovivian Let me know who will be working on this and I'll pair with them to provide improved error messages

**Comment by Onokaev** [comment]
Hey @RabebOthmani. I am picking this up

**Comment by RabebOthmani** [comment]
@Onokaev Let's add a check if snippet or permissions is failing to show due to a failed call, let's add a new error message to https://github.com/microsoftgraph/microsoft-graph-explorer-v4/blob/dev/src/messages/GE.json 
"Fetching code snippet failing": "Retry again",
"Fetching permissions failing": "Retry again",

We should show Snippet not available only when a code snippet is actually not available.

**Comment by Onokaev** [comment]
Hey @RabebOthmani. Can we reword this to 'Reload the tab' or something similar?

**Comment by RabebOthmani** [comment]
Is Retry again not ok?

**Comment by Onokaev** [comment]
I got feedback from a user at the office who said the grammar was off. They said we can have 'Retry' with a button for retrying the call or a different message that explains what happened during the call.
